### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by mdaum**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+## [1.0.0] - 2022-05-13
+
+- BREAKING: rename property `target` -> `targets` for BountySubmissionEntity
```
Comment (austinkelleher) [inline CHANGELOG.md:13]: I'm not sure whether anyone is using this property, but it will break existing queries if so.

**Review by austinkelleher — APPROVED** [review]
Approved with comment

File: CHANGELOG.md (hunk 1 above)
Comment (mdaum) [reply CHANGELOG.md:13]: I was able to confirm that there are no saved queries involving bugcrowd specifying this property

Comment (mdaum) [reply CHANGELOG.md:13]: nor were any made in past week

**Pull request merged**